src/pages/iou/request/step/IOURequestStepAmount.js

```diff
@@ -36,15 +36,17 @@ const defaultProps = {
 function IOURequestStepAmount({
     report,
     route: {
-        params: {iouType, reportID, transactionID, backTo},
+        params: {iouType, reportID, transactionID, backTo, currency: selectedCurrency},
     },
     transaction,
-    transaction: {currency},
+    transaction: {currency: originalCurrency},
 }) {
     const {translate} = useLocalize();
     const textInput = useRef(null);
     const focusTimeoutRef = useRef(null);
     const iouRequestType = getRequestType(transaction);
+    /* eslint-disable es/no-nullish-coalescing-operators */
+    const currency = selectedCurrency ?? originalCurrency;
```

Comment:
I don't think we should disable this lint rule

Reply:
@luacmartins Do you suggest to use `||` operator?

Reply:
I replaced the operator. Will you check again?